b_lambda_layer_common/source/python/util/http_call.py

```diff
@@ -37,6 +37,15 @@ def call(method, url, fields=None, headers=None, **urlopen_kw) -> HTTPResponse:
         """
         http = urllib3.PoolManager()
 
+        logger.info(
+            f'Calling endpoint: \n'
+            f'{method=}\n'
+            f'{url=}\n'
```

Comment:
`method` and `url` should be on the first line to compact the information

Reply:
Agreed.

Reply:
Btw. Missing dot at the end of the sentence.

Reply:
It's not a sentence anymore when everything is in new line. I don't see the need here for any dots.